cmd/kubeadm/app/apis/kubeadm/v1alpha1/conversion.go

```diff
@@ -123,3 +144,24 @@ func UpgradeAuthorizationModes(in *MasterConfiguration, out *kubeadm.MasterConfi
 		out.APIServerExtraArgs["authorization-mode"] = strings.Join(in.AuthorizationModes, ",")
 	}
 }
+
+func UpgradeBootstrapTokens(in *MasterConfiguration, out *kubeadm.MasterConfiguration) error {
+	if len(in.Token) == 0 {
+		return nil
+	}
+
+	bts, err := kubeadm.NewBootstrapTokenString(in.Token)
+	if err != nil {
+		return fmt.Errorf("can't parse .Token, and hence can't convert v1alpha1 API to a newer version: %v", err)
```

Comment:
Same thing on https://github.com/pkg/errors wrapping.

Reply:
I'll just leave this as-is as this is exactly the same code we have everywhere else in kubeadm. Error wrapping can be a more general, independent refactor later.

Reply:
sgtm